mvp/README.md

```diff
@@ -0,0 +1,19 @@
+
+**User-Centric** design enables the continuous delivery of value to users. This means that putting a good, functional, tool in the hands of users quickly is more valuable than delivering a perfect tool when it's too late.
```

Comment:
Keep this in mind, more important to have a good, functional tools delivered fast!